spec/workers/event_worker_spec.rb

```diff
@@ -14,7 +17,47 @@
   #   end.to change(described_class.jobs, :size).by(1)
   # end
 
-  pending 'can process an event'
-  pending 'can retry 3 times'
-  pending 'can clean up when errors are raised'
+  before do
+    stub_request(:post, 'https://myfakeoutbound.com/')
+      .with(headers: { 'Accept' => '*/*', 'User-Agent' => 'Ruby' })
+      .to_return(status: 200, body: { data: 'stubbed response' }.to_json, headers: {})
+  end
+
+  context 'Event worker' do
+    let(:event) { create :event }
+
+    it 'can process an event' do
+      expect(event.reload.completed).to eq false
+
+      EventWorker.new.perform event.id
+
+      expect(event.reload.completed).to eq true
+    end
+
+    it 'can retry 3 times' do
```

Comment:
I think I made this name but I don't like it anymore. I think we should get dynamic retries to work and then rename this to "can retry when errors are raised". I say this because we aren't actually testing if it can retry exactly 3 times. We are really testing a failed worker keeps the job in the queue.